CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.22 [2024-08-26]
+
+_Enhancements_
+
+- Added missing services to `Associated to` table in the VPC Security Group detail page. ([#352](https://github.com/turbot/steampipe-mod-aws-insights/pull/352)) (Thanks [@maxcorbin](https://github.com/maxcorbin) for the contribution!)
```

Comment:
@khushboo9024 could you please mention the services that were added to the VPC security group detail page?